src/utils/schemas.js

```diff
@@ -1,3 +1,5 @@
+import lodash from "lodash";
```

Comment:
It's better to load only lodash functions that we actually need in the module, not a whole lodash library.
https://stackoverflow.com/questions/35250500/correct-way-to-import-lodash

Reply:
good point! I adjusted it for all `lodash` imports in the repo

Reply:
@k0stik - the idea is good, provided that readability doesn't suffer, but let's deal with it in a separate task, pls see https://github.com/Exabyte-io/code.js/pull/40#discussion_r1089584173